frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_file_customize_location_view.dart

```diff
@@ -172,7 +173,10 @@ class _ChangeStoragePathButtonState extends State<_ChangeStoragePathButton> {
         onPressed: () async {
           // pick the new directory and reload app
           final path = await getIt<FilePickerService>().getDirectoryPath();
-          if (path == null || !mounted || widget.usingPath == path) {
+          if (path == null || widget.usingPath == path) {
+            return;
+          }
+          if (!mounted) {
```

Comment:
Why this change?
```suggestion
          if (!mounted || path == null || widget.usingPath == path) {
```

Reply:
Because Flutter 3.13 will raise an error if we combine the !mounted and other conditions.